package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.3",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10 <12"
+    "node": ">=10 <13"
```

Comment:
This I did not submit intentionally, will remove before the final PR to the epic branch